modyn/trainer_server/internal/trainer/remote_downsamplers/irreducible_loss_producer.py

```diff
@@ -0,0 +1,104 @@
+import io
```

Comment:
Can we restructure the remote_downsamplers directory to have a subdirectory rholoss, where this and the remote_rs2_downsampler files go? in __init__.py of remote_downsamplers we can then import the RS2 Downsampler such that 

from trainer.remote_downsamplers import RS2Downsampler

is still possible. This avoids that this file lives on the same level as the other downsamplers

Reply:
I think I will only move this `irreducible_loss_producer.py` file to the subdirectory, because for consistency we also have the `deepcore_utils` subdirectory which holds all the utilities for deepcore-based downsamplers, which reside out of the subdirectory.